testing/endtoend/components/beacon_node.go

```diff
@@ -257,6 +257,7 @@ func (node *BeaconNode) Start(ctx context.Context) error {
 		fmt.Sprintf("--%s=%s", flags.ExecutionJWTSecretFlag.Name, jwtPath),
 		fmt.Sprintf("--%s=%d", flags.MinSyncPeers.Name, 1),
 		fmt.Sprintf("--%s=%d", cmdshared.P2PUDPPort.Name, e2e.TestParams.Ports.PrysmBeaconNodeUDPPort+index),
+		fmt.Sprintf("--%s=%d", cmdshared.P2PQUICPort.Name, e2e.TestParams.Ports.PrysmBeaconNodeQUICPort+index),
```

Comment:
We should also add the `enable-quic` flag here, so that every prysm node is running with it enabled in e2e

Reply:
Yes, it was implicit in the initial PR, since QUIC was enabled by default.
Now it's not.

Reply:
Fixed in 1f0d84d0094f5dd58743e4316fa4be4043679614.